src/main/java/uk/gov/hmcts/cmc/claimstore/documents/DefendantResponseReceiptService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.pdf.DocumentTemplates;
+import uk.gov.hmcts.cmc.claimstore.documents.content.DefendantResponseContentProvider;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.cmc.pdf.service.client.PDFServiceClient;
+
+import static java.util.Objects.requireNonNull;
+
+@Service
+public class DefendantResponseReceiptService {
+
+    private final DefendantResponseContentProvider contentProvider;
+    private final DocumentTemplates documentTemplates;
+    private final PDFServiceClient pdfServiceClient;
+
+    @Autowired
+    public DefendantResponseReceiptService(
+        final DefendantResponseContentProvider contentProvider,
+        DocumentTemplates documentTemplates, final PDFServiceClient pdfServiceClient
```

Comment:
Please make one argument per line.

Reply:
This has already been changed, just not updated on this.